### PR TITLE
Updated methods/stat_tracker/spec_helper

### PR DESCRIPTION
**Pull request by sethverrill**
Adjusted team name method to be a little more dynamic.
Added methods for avg goals and avg away goals
Adjusted spec to reflect new data.

### CODE REVIEW

File: lib/team.rb [hunk 1]
```diff
@@ -6,6 +6,6 @@ class Team
   def initialize(data)
     @team_id = data["team_id"]
     @teamName = data["teamName"]
-    @staduim = data["Stadium"]
+    @stadium = data["Stadium"]
```
Comment (Jonathan-Atkins) [inline lib/team.rb:9]: noice

**Review by Jonathan-Atkins — COMMENTED** [review]
looks good

File: lib/league_statistics.rb [hunk 2]
```diff
@@ -17,7 +17,11 @@ def count_of_teams
   end
 
   def best_offense
-    team_name(team_avg_goals.max_by { |team_id, avg_goals| avg_goals}[0])
+    team_name(team_avg_goals.max_by { |team_id, avg_goals| avg_goals }[0])
+  end
+
+  def worst_offense
+    team_name(team_avg_goals.min_by { |team_id, avg_goals| avg_goals }[0])
```
Comment (MiTOBrien) [inline lib/league_statistics.rb]: Nice clean code.

**Comment by MiTOBrien** [comment]
Code looks good.  Merging.

**Pull request merged**